.readthedocs.yml

```diff
@@ -23,6 +23,10 @@ python:
   install:
     - requirements: doc/requirements.txt
     - requirements: qchem/requirements.txt
+    - method: pip
+      path: pyscf
+    - method: pip
+      path: openfermionpyscf
```

Comment:
Seems like these read the docs treats these as local paths. A hot fix could be to create some `extra_requirements.txt` file (though not sure on the best practices there). Haven't come across a solution for pulling the packages from PyPi.

Reply:
I was just looking at this and wondering how it could be solved. :thinking: An `extra_requirements.txt` file would probably work, but doesn't seem like the prettiest solution. It seems like something that they should support here; strange that they don't (as far as I can tell).

Reply:
I went with that solution for now anyway. Thanks @antalszava! Couldn't think of anything better. Perhaps @josh146 would have other ideas/inputs?